.travis.yml

```diff
@@ -1,61 +1,32 @@
 language: generic
-notifications:
-  email:
-    recipients:
-      - kubernetes-sig-service-catalog-alerts@googlegroups.com
-env:
-  - GO_VERSION=1.9
-  - GO_VERSION=rc
 matrix:
   allow_failures:
     - env: GO_VERSION=rc
   fast_finish: true
 sudo: required
 services:
 - docker
-cache:
-  directories:
-    # ${GOPATH}/pkg is mounted in DOCKER_CMD in Makefile
-    - .pkg
 stages:
   - test
-  - name: xbuild # Test all of our supported platform builds for master builds
-    if: type != pull_request
   - name: deploy
     if: type != pull_request
-before_install:
-  - |
-      if [[ -z "$TRAVIS_COMMIT_RANGE" ]]; then
-          # Builds triggered by initial commit of a new branch.
-          DOCS_ONLY=0
-      else
-          DOCS_REGEX='(OWNERS|LICENSE)|(\.md$)|(^docs/)'
-          [[ -z "$(git diff --name-only $TRAVIS_COMMIT_RANGE | grep -vE $DOCS_REGEX)" ]]
-          DOCS_ONLY=$?
-      fi
-# Test
-script:
-  - |
-    if (( $DOCS_ONLY == 0 )); then
-      echo "Running verify-docs"
-      make verify-docs
-    else
-      echo "Running full build"
-      make verify build build-integration build-e2e test images svcat-all
-    fi
 jobs:
   include:
-    # Test is implicit from the build matrix
-    # Cross Build
-    - stage: xbuild
-      script: make images-all
+    # Build
+    - stage: test
+      script: bash -c "env"
+      env: GO_VERSION=1.9
+    # Go RC Check (allowed to fail)
+    - stage: test
+      script: bash -c "env"
+      env: GO_VERSION=rc
```

Comment:
while we're in here, we should probably change this to 1.10 or remove it, as the rc tag isn't being updated anymore. travis has `tip` but the docker hub release does not.

Reply:
Good call! 💯